site/homepage/content/release-notes.md

```diff
@@ -11,6 +11,13 @@ title = "Release Notes"
   This has the advantage of saving the network hop to the Device Connection service.
   Refer to the protocol adapter Admin Guides for details.
 
+### Fixes & Enhancements
+
+* The service base classes for implementing device registry namely `CredentialsManagementService`,
+  `CredentialsService`, `DeviceConnectionService`, `DeviceManagementService`, `RegistrationService`,
+  `TenantManagementService`and `TenantService` have been refactored now to return a Vertx future instead of 
+   taking an asynchronous handler as an argument.
```

Comment:
I think that fits better in the "API Changes" section.

I would omit the "for implementing device registry namely" part and would use 
"refactored to now return a Vert.x `Future` instead of taking a `Handler` as an argument"
in the last part.
And I would prepend "The methods of [the service base classes]".

Reply:
@calohmn I have modified it and could you take a look below before I push the changes?

The methods of the service base classes `CredentialsManagementService`, `CredentialsService`, `DeviceConnectionService`, `DeviceManagementService`, `RegistrationService`, `TenantManagementService` and  `TenantService` are refactored now to return a Vert.x Future instead of taking a Handler as an argument.

Reply:
How about: "have been refactored to now return"?

Reply:
Sounds good. I have modified and pushed the changes.